heron/ckptmgr/src/java/com/twitter/heron/ckptmgr/CheckpointManagerConfig.java

```diff
@@ -0,0 +1,79 @@
+// Copyright 2016 Twitter. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.twitter.heron.ckptmgr;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.twitter.heron.common.config.ConfigReader;
+
+public class CheckpointManagerConfig {
+  public static final String STORAGE_CONFIG = "heron.statefulstorage.config";
```

Comment:
i don't think its a good idea making this as complicated as SystemConfig. It looks very heavy weight for this purpose